practice-one/styles/components/nav.css

```diff
@@ -1,18 +1,22 @@
 .list {
```

Comment:
Just **list** is not good enough to understand and can imagine where is this section will show on the design